Bug, Build fails to kover on just compile

This appears to be a configuration issue with “kover”.
Running is fine, but builds like “Make Project” fail.

<img width="1242" alt="a" src="https://github.com/user-attachments/assets/cb44835e-7448-4e0b-9a44-608107d6f639">

I don't know kover, so I'm hoping someone can help me with this.

```
Build file '/Users/pluu/SampleProject/conference-app-2024/app-android/build.gradle.kts' line: 4

An exception occurred applying plugin request [id: 'droidkaigi.primitive.kover.entrypoint']
> Failed to apply plugin 'org.jetbrains.kotlinx.kover'.
   > Cannot run Project.afterEvaluate(Action) when the project is already evaluated.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
For more on this, please refer to https://docs.gradle.org/8.9/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
BUILD FAILED in 3s
5 actionable tasks: 1 executed, 4 up-to-date
Configuration cache entry stored.
```

--- a/build-logic/src/main/kotlin/io/github/droidkaigi/confsched/primitive/KoverEntryPointPlugin.kt
+++ b/build-logic/src/main/kotlin/io/github/droidkaigi/confsched/primitive/KoverEntryPointPlugin.kt
@@ -13,7 +13,10 @@ class KoverEntryPointPlugin : Plugin<Project> {
 
             rootProject.subprojects {
                 if (this@subprojects.name == target.name) return@subprojects
-                this@subprojects.pluginManager.apply(koverPlugin)
+                // https://github.com/DroidKaigi/conference-app-2024/issues/485#issuecomment-2304251937
+                this@subprojects.beforeEvaluate { // wrap with beforeEvaluate
+                    this@subprojects.pluginManager.apply(koverPlugin)
+                }
                 target.dependencies.add("kover", this@subprojects)
             }
 

